package.json

```diff
@@ -70,53 +71,10 @@
     "extend-error": "git://github.com/ianwremmel/extend-error.git",
     "gm": "~1.18.1",
     "grunt-file-exists": "^0.1.4",
-    "lodash.assign": "^3.1.0",
```

Comment:
Well, there's a good chance the breaking changes between 3 and 4 will be a problem. And it'll grow the bundle size quite a bit because of pulling in all of lodash